CHANGES.md

```diff
@@ -1,6 +1,7 @@
-# Unreleased
+# 2.13.0 (2020-04-14)
 
 - [FIXED] Correctly raise exceptions from `create_database` calls.
+- [FIXED] Fix "DeprecationWarning" from "collections"
```

Comment:
nits:
* needs an ending period/full stop `.`
* I'd prefer to see DeprecationWarning and collections, marked up as monospace rather than double-quoted.